lib/Dialect/TritonNvidiaGPU/Transforms/MaterializeLoadStore.cpp

```diff
@@ -202,6 +219,90 @@ void MaterializeLoadStorePass::materializeStoreTilePtr(
   store->erase();
 }
 
+SmallVector<Value>
+MaterializeLoadStorePass::immutableMbarrierAlloc(Operation *entryOp,
+                                                 unsigned loadNum) {
+  SmallVector<Value> barrierLists;
+  OpBuilder builder(entryOp);
+  Location loc = entryOp->getLoc();
+  builder.setInsertionPoint(entryOp);
```

Comment:
Since you create the builder with `OpBuilder builder(entryOp);`
Then it's unnecessary to additionally set the insertion point.

Reply:
All right, modified.